### ISSUE TITLE
Replace print statement with CodegenLogger.log

### ISSUE THREAD

**Issue by hiltonc** [apollographql/apollo-ios#2346]
## Bug report

I'm using `ApolloCodegenLib` to script codegen, and want to reduce the verbosity of the output. I found that I could do `CodegenLogger.level = .error`, but I still see this output:

```
SHASUM of downloaded file: 479e1c12adf631c43c093fdd9dd8e5c8eed33cb15e39de7165ce6f5e7f759904
```

I've traced it to https://github.com/apollographql/apollo-ios/blob/55c5db823811d1d1168adcbe6e98a862583c215d/Sources/ApolloCodegenLib/CLIExtractor.swift#L138, I think it just needs to be changed to use `CodegenLogger.log` instead of `print`.

## Versions

Please fill in the versions you're currently using: 

- `apollo-ios` SDK version: 0.39.0 (but I see this on the tip of main as well)
- Xcode version: 14.0.0 beta 2
- Swift version: 5.6
- Package manager: SPM

## Steps to reproduce

Script `ApolloCodegenLib`, set `CodegenLogger.level = .error`, download a schema. "SHASUM of downloaded file..." continues to print.

**Comment by calvincestari** [comment]
Yes, this seems reasonable. Would you be able to submit a PR making the change?
